biogeochem/FatesCohortMod.F90

```diff
@@ -1002,6 +1005,39 @@ end function CanUpperUnder
    
     !===========================================================================
 
+    function SumMortForHistory(this, per_year) result(mort_sum)
+      !
+      ! DESCRIPTION:
+      ! Sum the various cohort-level mortality variables for saving to history.
+      !
+
+      ! ARGUMENTS:
+      class(fates_cohort_type) :: this ! current cohort of interest
+      logical                  :: per_year
+      !
+      ! VARIABLES
+      real(r8) :: mort_natural
+      real(r8) :: mort_logging
+      real(r8) :: mort_sum
+
```

Comment:
Lets double check the units on the natural versus the logging mortality, I seem to remember they were different, such as their usage here:

https://github.com/NGEET/fates/blob/main/biogeochem/EDMortalityFunctionsMod.F90#L347-L354

Reply:
No, the problem is that one is in # individuals and the other is fraction.

- [ ] Fix this.